physical/consul/consul.go

```diff
@@ -224,13 +229,47 @@ func SetupSecureTLS(ctx context.Context, consulConf *api.Config, conf map[string
 	return nil
 }
 
+// ExpandedCapabilitiesAvailable tests to see if Consul has KVGetOrEmpty and 128 entries per transaction available
+func (c *ConsulBackend) ExpandedCapabilitiesAvailable(ctx context.Context) bool {
```

Comment:
This name is sort of generic. How do you feel about `HasKVGetOrEmptyAnd128Entries` or the specific API version that we're checking against instead?

Reply:
I made the name generic on purpose, thinking it was better to do that than spell out the implementation details in the name. Maybe we want to change how it does this check in the future?

Reply:
That works for me. I guess my hesitation with this name was "ExpandedCapabilities" doesn't really indicate "backend TxnOps" super clearly, but naming is hard!